src/main.rs

```diff
@@ -40,29 +61,51 @@ fn main() -> DynResult<()> {
             &format!("git@github.com:{}", &repo),
             repo_path.as_os_str().to_str().unwrap(),
         ])
+        .stderr(Stdio::inherit())
+        .stdout(Stdio::inherit())
         .output()
         .expect("Error cloning repo");
-    eprintln!("Cloned {} to {:?}", raw_url, repo_path);
+    info!("Cloned {} to {:?}", raw_url, repo_path);
 
-    Command::new("git")
-        .current_dir(&repo_path)
-        .args(["fetch", "origin", &format!("pull/{}/head:GHDOC", number)])
-        .output()
-        .expect("Error fetching PR");
-    Command::new("git")
-        .current_dir(&repo_path)
-        .args(["checkout", "GHDOC"])
-        .output()
-        .expect("Error checking out PR branch");
-    eprintln!("Checked out PR {}", number);
+    if let Some(number) = number {
+        Command::new("git")
+            .current_dir(&repo_path)
+            .args(["fetch", "origin", &format!("pull/{}/head:GHDOC", number)])
+            .stderr(Stdio::inherit())
+            .stdout(Stdio::inherit())
+            .output()
+            .expect("Error fetching PR");
+        Command::new("git")
+            .current_dir(&repo_path)
+            .args(["checkout", "GHDOC"])
+            .stderr(Stdio::inherit())
```

Comment:
so stderr gets re-routed to stdout?  Why not just to stderr

Reply:
It's having the child process inherit the stderr file descriptor from the parent, likewise for stdout.

I had to read the docs a few times to figure that out: https://doc.rust-lang.org/std/process/struct.Stdio.html#method.inherit

Reply:
So stderr and stdout from the spawned processes both transperently show up on the user end so any `git` error messages or `cargo doc` error messages / logs are visible.